garage/envs/box2d/box2d_env.py

```diff
@@ -110,7 +112,7 @@ def action_space(self):
             [control.ctrllimit[0] for control in self.extra_data.controls])
         ub = np.array(
             [control.ctrllimit[1] for control in self.extra_data.controls])
-        return gym.spaces.Box(lb, ub, dtype=np.float32)
```

Comment:
@ryanjulian exactly, I believe this is the same issue as you mentioned in the PR chat.